### ISSUE TITLE
Constants.xhtml contains illegal XML characters

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#5523]
|<img alt="greezybacon's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">| greezybacon reported|
|-|-|
|Trac ID|trac#5523|
|Type|defect|
|Created|2012-03-12 02:05:47Z|

doc/core/howto/constants.xhtml contains bare ampersand (&) characters. In XML, &amp; should be used to represent an ampersand character.

The attached patch fixes the issue.

Attachments:

* [constants.xhtml.patch](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/ff8/ff8dba29c5059915cdc37fc87f8a9d1835f70897/15f23f0fcb5d61adc342e23dba07788b0df8f404.patch) (933 bytes) - added by greezybacon on 2012-03-12 02:06:18Z - Patch to use XML entities for the ampersand characters

<details><summary>Searchable metadata</summary>

```
trac-id__5523 5523
type__defect defect
reporter__greezybacon greezybacon
priority__normal normal
milestone__ 
branch__ 
branch_author__ 
status__closed closed
resolution__fixed fixed
component__core core
keywords__documentation documentation
time__1331517947000000 1331517947000000
changetime__1331540644000000 1331540644000000
version__None None
owner__glyph glyph

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="glyph's avatar" src="https://avatars.githubusercontent.com/u/716529?s=50" width="50" height="50">](https://github.com/glyph)<a name="note_2"></a>|@glyph set owner to @glyph|
|-|-|

Looks great.  Thanks!  I'll merge this momentarily.

**Comment by twisted-trac** [comment]
|[<img alt="glyph's avatar" src="https://avatars.githubusercontent.com/u/716529?s=50" width="50" height="50">](https://github.com/glyph)<a name="note_3"></a>|@glyph set status to `closed`|
|-|-|

(In [33686]) Correct invalid XML in constants documentation.

Fixes: [#5523](https://github.com/twisted/twisted/issues/5523)

Author: greezybacon

Reviewer: glyph

This change replaces some stray `&`'s with the XML-correct `&amp;`.
